platform/web/emscripten_helpers.py

```diff
@@ -115,6 +118,10 @@ def create_template_zip(env, js, wasm, worker, side):
     )
 
 
+def get_template_zip_path(env):
+    return "#bin/.web_zip"
```

Comment:
I'm assuming the reason to make this a function was so you can use a different path for the two types of builds? This seems to be missing.

Reply:
Initially, it was, that function had logic. But now it doesn't have. But it's better now even if it's returning a static string because there was 2 magic strings in the code otherwise.